frontend/GameGUI.py

```diff
@@ -13,21 +13,57 @@ def __init__(self):
         self.screen = pg.display.set_mode((WIN_SIZE, WIN_SIZE + self.info_height))
         self.clock = pg.time.Clock()
         # window title
-        pg.display.set_caption("Nine-Mens-Morris")
-        self.board_size = 7
-        self.total_pieces = 9
+        
+        self.font = pg.font.Font(None, 34)
+        self.title_font = pg.font.Font(None, 48)
+
+    def finish_init(self, size, pieces):
+        cap = "-Mens-Morris"
+        if pieces == 6:
+            self.title = "Six"
+        elif pieces == 9:
+            self.title = "Nine"
+        else:
+            self.title = "Twelve"
+        pg.display.set_caption(self.title + cap)
+        self.board_size = size
+        self.total_pieces = pieces
         self.game_manager = GameManager(self.board_size, self.total_pieces)
-        self.board_gui = BoardGUI(
-            self, WIN_SIZE, self.board_size, self.game_manager
-        )
+        self.board_gui = BoardGUI(self, WIN_SIZE, self.board_size, self.game_manager)
+
+    def game_picker_screen(self):
```

Comment:
I hate this, lol. Will implement differently once we have all changes merged into main, because Suryansh's replay also has a start screen like this.